about-us.html

```diff
@@ -10,32 +15,32 @@
 
 <body>
     <header class="page-header">
-        <a class="main-logo"><img src="img/logo.svg" alt="binabox-logo">
-            <span class="logo-text">BINABOX</span></a>
         <nav class="top-nav">
+            <a class="main-logo"><img src="images\logo\binaboxlogo36x40.svg" alt="binabox logo" width="188" height="42"></a>
             <ul class="nav-list">
                 <li class="nav-item"><a href="index.html">HOME</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
-                <li class="nav-item"><a href="about.html">ABOUT</a>
+                <li class="nav-item"><a href="about-us.html">ABOUT</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
                 <li class="nav-item"><a href="roadmap.html">ROADMAP</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
                 <li class="nav-item"><a href="collection.html">COLLECTION</a>
                 </li>
                 <li class="nav-item"><a href="faqs.html">FAQS</a>
+                    <button class="dropbtn">drop arrow</button>
                 </li>
                 <li class="nav-item"><a href="pages.html">PAGES</a>
+                    <button class="dropbtn">drop arrow</button>
             </ul>
             <div class="nav-buttons">
-                <button class="toggle-button" onclick="toggle()">
-                    <img id="sun-icon" src="light-mode-icon.png" alt="Toggle Light/Dark Mode">
-                </button>
-                <button type="button" class="discord-button">
-                    <img src="#" alt="discord-logo"> DISCORD
-                </button>
-                <button type="button" class="connect-button">
-                    <img src="#" alt="connect-logo"> CONNECT
+                <button class="toggle-button" onclick="toggle()">toggle
+                    <img id="sun-icon" src="images\buttons\switch.svg" width="22.5" height="22.5" alt="">

```

Comment:
This is also a non-content image, and there’s no need to include it in the HTML.

Non-content images, such as decorative elements, are better placed in CSS rather than HTML. This helps separate content from presentation, improving both SEO and accessibility. Including such images directly in HTML can clutter your code and make it harder to maintain. Read more about handling decorative images.

Check all images.